learn_js/part_6.7/script67.js

```diff
@@ -0,0 +1,58 @@
+//my task
+
+let d = setInterval(() => console.log("da"), 1000);
+
+setTimeout(() => {
+    clearInterval(d);
+    console.log("stop")
+}, 5000);
+
+//my task recursion
+
+let timeDel = setTimeout(function aa() {
+
+    console.log("tick");
+    timeDel = setTimeout(aa, 1000);
+
+}, 4000);// delay before first call
+
+//task 1
+function printNumbers(from = 5000, to = 10000) {
+    if (from < to) {
+
+        setTimeout(() => {
+
+            let goInt = setInterval(() => { console.log("ky-ky  " + new Date) }, 1000);
+
+            setTimeout(() => clearInterval(goInt), to - from);
+
+        }, from);
+
```

Comment:
Don't forget about codestyle!

Reply:
well, but it`s not project and so i see better